src/code/game.c

```diff
@@ -89,8 +89,11 @@ void GameState_Draw(GameState* gamestate, GraphicsContext* gfxCtx) {
     Graph_BranchDlist(_polyOpa, nextDisplayList);
     gfxCtx->polyOpa.p = nextDisplayList;
 
-lblUnk:; // Label prevents reordering, if(1) around the above block don't seem to help unlike in OoT
-    func_800E9F78(gfxCtx);
+    // Block prevents reordering, if(1) around the above block don't seem to help unlike in OoT
+    {
+        s32 pad;
+        func_800E9F78(gfxCtx);
```

Comment:
```suggestion
        s32 pad;
        
        func_800E9F78(gfxCtx);
```

Reply:
Whoops, I was so excited about this trick that I forgot to do this. I'll fix this (and all the others below), thanks

Reply:
This function is also missing DISPS macros, from the look of it, which might also affect things. I think #89 fixes that, but I wouldn't wait for it.

Reply:
I got it matching with the DISPS macros; I had to delete the unused variables. I also cleaned it up in the same way that #89 did, but I can revert that cleanup if you'd rather it go in with lucas's PR.